src/styles/common.less

```diff
@@ -356,13 +362,13 @@ p {
 }
 
 a {
-  color: #337ab7;//bootstrap 4 changed its defaul to 007bff, change it back here
+  color: @sb-cerulean-900;//bootstrap 4 changed its defaul to 007bff, change it back here
 }
 
 .alert.show:not(.alert-danger):not(.alert-success){ //in bootstrap3, this is handled via alert-info, but the alert dialog is not the same with bootstrap4
-  color: #31708f;
-  background-color: #d9edf7;
-  border-color: #bce8f1;
+  color: @sb-blue-800;
```

Comment:
This basically is black text on a dark gray background. This cannot be right.

Reply:
This does show as a 9.44 Contrast Ratio which is actually higher than our text color on the white background (5.77 for reference). Still a good catch, and something we should be aware we are changing.